pipelines/prontuarios/std/formatters/generic/patient.py

```diff
@@ -0,0 +1,280 @@
+        str: Normalized and validated data
+    """
+    if datetime is None:
+        return None
+    else:
+        clean_datetime = re.sub(r" .*", "", datetime)
+        clean_datetime = re.sub(r"T.*", "", clean_datetime)
+        clean_datetime = pd.to_datetime(clean_datetime, format="%Y-%m-%d", errors="coerce")
+        clean_datetime = str(clean_datetime.date()) if pd.isna(clean_datetime) is False else None
+        return clean_datetime
+
+
+def clean_name_fields(name: str) -> str:
+    """
+    clean name
+
+    Args:
+        name (str): Name field to be standardized
+    Returns:
+        str: Standardized name string
+    """
+    if name is None:
+        return
+    else:
+
+        name = re.sub(r"\( *(ALEGAD[O|A]) *\)", "", name)
+        name = re.sub(r"[´`'.]", "", name)
+        name = re.sub(r"Ã§", "C", name)
+        name = re.sub(r'[!"#$%&\'()*+,-\/:;<=>?@[\\\]^_`{|}~]', "", name)
+        name = name.replace("\t", "")
+        name = name.strip()
+        name = re.sub("(SEM INFO)|(DECLAR)|(PAC CHEGOU COM OS BOMBEIROS)", "", name)
+        name = unidecode(name)
+
+        if bool(re.search(r"[^\w ]", name)) | len(name.split()) < 2:
+            return ""  # mudar quando Pedro mudar api para nome nao ser obg
+        else:
+            return name
+
+
+def dic_cns_value(valor: str, is_main: bool) -> dict:
+    """
+    Format register dictionary of CNS value
+    Args:
+        valor (str): CNS value
+        is_main (bool): Flag if CNS value is main patient CNS
+    Returns:
+        dict: CNS info dictionary
+    """
+    cns = CNS()
+    valor = re.sub("[^0-9]", "", valor)
+    if valor is None:
+        return
+    elif cns.validate(valor):
+        return {"value": valor, "is_main": is_main}  # o primeiro da lista é o main
+    else:
+        return
+
+
+def format_address(pp_type: str, pp_name: str, number: str, compl: str) -> str:
+    """
+    Returns full line address combining columns
+
+    Args:
+    pp_type (str): Type of public place
+    pp_name (str): Public place name
+    number (str):
+
+    Returns:
+        str: Full adress line
+    """
+    logrado_type = pp_type if pp_type is not None else ""
+    logrado = pp_name if pp_name is not None else ""
+    number = f", {number}" if number is not None else ""
+    compl = compl if compl is not None else ""
+
+    return f"{logrado_type} {logrado}{number} {compl}"
+
+
+def clean_postal_code_info(data: dict) -> dict:
+    """
+    Validade cep info
+
+    Args:
+        data (dict) : Individual data record
+
+    Returns:
+        data (dict) : Individual data record standardized
+    """
+    if data["end_cep"] is None:
+        data["postal_code"] = None
+        return data
+    else:
+        data["end_cep"] = re.sub(r"[^0-9]", "", data["end_cep"])
+        if len(data["end_cep"]) != 8:
+            data["postal_code"] = None
+            return data
+        else:
+            data["postal_code"] = data["end_cep"]  # [0:5] + '-' data['end_cep'][5:9]
```

Comment:
Esse comentário representa oq? Se for uma melhoria, podemos criar uma atividade no Clickup